doc/api/async_context.md

```diff
@@ -215,7 +215,7 @@ added:
  - v12.17.0
 -->
 
-> Stability: 1 - Experimental
+> Stability: 2 - Stable
```

Comment:
I think we should mention the version when it was moved from experimental to stable in the YAML section above.
see e.g. `AsyncLocalStorage.snapshot()` as reference.